src/.nuget/Microsoft.NETCore.Crossgen/Microsoft.NETCore.Crossgen.pkgproj

```diff
@@ -0,0 +1,38 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="12.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <Import Project="$([MSBuild]::GetPathOfFileAbove(Directory.Build.props))" />
+
+  <PropertyGroup Condition="'$(PackageTargetRuntime)' == ''">
+    <IsLineupPackage Condition="'$(IsLineupPackage)' == ''">true</IsLineupPackage>
+  </PropertyGroup>
+
+  <Import Condition="'$(_packageTargetOSGroup)' != ''" Project="$(MSBuildThisFileDirectory)runtime.$(_packageTargetOSGroup).$(MSBuildProjectName).props" />
+
+  <Import Project="$([MSBuild]::GetPathOfFileAbove(Directory.Build.targets))" />
+
+  <PropertyGroup>
+    <!-- We're publishing crossgen2 as framework dependent application in the runtime package for now since we only have
+        two simple scenarios we want to support in the short term: win_x64 to win_x64 and linux_x64 to linux_x64 compilations.
+        Once we have more complex targets, especially cross-platform and cross-architecture, crossgen2 should move to its own
+        package, and become a self-contained package.
+    -->
+    <CpaotCrossgenRuntimeConfigContents>
+{
+  "runtimeOptions": {
+    "tfm": "netcoreapp3.0",
+    "rollForward": "major",
+    "framework": {
+      "name": "Microsoft.NETCore.App",
+      "version": "5.0"
```

Comment:
This is temporary, only for dotnet5, In the future, we'll build crossgen2 as a self-contained application, and we won't need to deal with this config file.

Reply:
I'm surprised this works. This is supposed to be semantic version. 5.0 is greater than any actual 5.0 runtime available. Still think hand-crafting this is bad and you should generate it.
cc @vitek-karas

Reply:
We can't generate a proper runtimeconfig right now given that we're publishing self-contained. Even if we set `GenerateRuntimeConfigurationFiles` to true, it would still generate an empty runtimeconfig that won't be usable when running as a FD app.
It's possible to package the compiler as a self-contained application, but for v5 scenarios this would be a waste and would increase the download sizes for no good reason (we don't have cross-arch/plat compilation scenarios for v5, so it's easy to just host the compiler using the installed SDK).

Reply:
So you're publishing self-contained and manually extracting out FDD? hacky if so. Can you publish FDD?

Reply:
I suggest that you remove the rollForward, and pick an actual version that is lower than all 5.0 builds, say `5.0.0-alpha1.0`. That is closer to what you would get if you were targeting 5.0 and letting the SDK gen the config.

Reply:
I suggest that if you cannot just publish FDD.

Reply:
I don't even know why 5.0 works here with what you have.

Reply:
> pick an actual version that is lower than all 5.0 builds, say 5.0.0-alpha1.0

Sounds good. Even if we published FDD right now (which also means we'll have to build crossgen2 twice), the runtimeconfig will have 3.0.* as a version, which still won't work with dotnet 5 unless we have rollForward.